### PR TITLE
Add note about venv packages (SOC-9958)

### PR DESCRIPTION
**Pull request by eapodaca**
* Add note that describes the venv- packages and how to determine openstack release.

### CODE REVIEW

**Comment by jeremy-moffitt** [comment]
whats interesting is that the <uri> element is not automatically clickable... wonder if @csymons-suse  has any suggestions on that (doesnt really matter)

**Comment by jeremy-moffitt** [comment]
I'm not clear why Travis is failing, it doesnt like the commit message but looks to me like a library get timeout https://travis-ci.org/SUSE-Cloud/doc-cloud/jobs/574511121

File: xml/day2_ui_documentation.xml [hunk 1]
```diff
@@ -163,6 +163,14 @@
      </imageobject>
     </mediaobject>
    </figure>
+    <note>
+      <para>
+        Packages with the <filename>venv-</filename> prefix denote the version of
+        the specific openstack package that is deployed. The OpenStack releases
+        (<uri>https://releases.openstack.org/</uri>) page can be refrenced to
```
Comment (csymons-suse) [inline xml/day2_ui_documentation.xml]: Packages with the <literal>venv-</literal> prefix (the filename tag is for files or directories).

For the linked sentence I suggest...
The release name can be determined from the <link xlink:href="https://releases.openstack.org/">OpenStack Releases</link> page.

Comment (csymons-suse) [reply xml/day2_ui_documentation.xml]: Be nice to see the actual text...
`The release name can be determined from the <link xlink:href="https://releases.openstack.org/">OpenStack Releases</link> page.`

Comment (csymons-suse) [reply xml/day2_ui_documentation.xml]: and the tags...GitHub is picky. use `literal` instead of `filename`.

**Comment by csymons-suse** [comment]
> I'm not clear why Travis is failing [travis-ci.org/SUSE-Cloud/doc-cloud/jobs/574511121](https://travis-ci.org/SUSE-Cloud/doc-cloud/jobs/574511121)
> 
> > Commit [df3c6a5](https://github.com/SUSE-Cloud/doc-cloud/commit/df3c6a5bb9657a9c91f94bd5450ca82f748c3d67):
> > 3: B6 Body message is missing

Well it obviously has a body message...maybe eliminate the bullet mark. Just raw ASCII text.

**Comment by csymons-suse** [comment]
Another detail...when this validates properly, it should be cherry-picked (`cherry-pick -x`) to the maintenance/cloud_9 branch.

**Comment by csymons-suse** [comment]
replaced by 
https://github.com/SUSE-Cloud/doc-cloud/pull/1131

**Pull request closed without merging**